tools/windows-node-installer/test/e2e/azure_test.go

```diff
@@ -333,13 +333,19 @@ func createHostFile(ip, password string) (string, error) {
 	}
 	defer hostFile.Close()
 
+	// Give a loop back ip as internal ip, this would never show up as
+	// private ip for any cloud provider. This is a dummy value for testing
+	// purposes
+	// TODO:Remove this and get the ip address from the cloud provider
+	// 		using instance ID from the node object
+	loopbackIP := "127.0.0.1"
```

Comment:
Do we even need this here? I think the test will pass the ping test even without this. Or am I mistaken?

Reply:
I added it for the sake of consistency. Actually this entire test can be removed with a port open check.